config/i18n-tasks.yml

```diff
@@ -317,8 +317,9 @@ ignore_unused:
   - decidim.assemblies.admin.assembly_members.form.explanation
   - decidim.assemblies.admin.assembly_members.form.image_guide
   - decidim.assemblies.admin.assembly_members.form.non_user_avatar_help
+  - decidim.blogs.admin_log.*
   - decidim.templates.admin_log.*
-
+  
```

Comment:
Could you please remove the trailing whitespace here? CodeClimate is complaining about this.